deploy/13-deploy-wstETH-weETH-oracles.ts

```diff
@@ -0,0 +1,45 @@
+import hre from "hardhat";
+import { DeployFunction } from "hardhat-deploy/dist/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { ADDRESSES } from "../helpers/deploymentConfig";
+
+const func: DeployFunction = async function ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const proxyOwnerAddress = ADDRESSES[network.name].timelock;
+  const { WETH, wstETH, weETH } = ADDRESSES[network.name];
+
+  const resilientOracle = await hre.ethers.getContract("ResilientOracle");
+  const chainlinkOracle = await hre.ethers.getContract("ChainlinkOracle");
+
+  await deploy("wstETHOneJumpChainlinkOracle", {
+    contract: "OneJumpOracle",
+    from: deployer,
+    log: true,
+    deterministicDeployment: false,
+    args: [wstETH, WETH, resilientOracle.address, chainlinkOracle.address],
+    proxy: {
+      owner: proxyOwnerAddress,
+      proxyContract: "OptimizedTransparentProxy",
```

Comment:
We are using `OptimizedTransparentUpgradeableProxy` for the rest of the deployments in the oracle repo

Reply:
Deployed oracles with OptimizedTransparentUpgradeableProxy, a9966bc35c905f1bc84009af4f8a728ba2e1d98e